source/game/kart/KartParamFileManager.cc

```diff
@@ -96,7 +96,7 @@ EGG::RamStream KartParamFileManager::getHitboxStream(Vehicle vehicle) const {
 }
 
 EGG::RamStream KartParamFileManager::getBikeDispParamsStream(Vehicle vehicle) const {
-    if (vehicle < Vehicle::Standard_Bike_S && vehicle >= Vehicle::Max) {
+    if (vehicle < Vehicle::Standard_Kart_S || vehicle >= Vehicle::Max) {
```

Comment:
If a kart attempts to load the bike display parameters, this change will cause a segfault.

Reply:
reverted.